backend/audit/intake_to_dissemination.py

```diff
@@ -17,45 +17,62 @@
 logger = logging.getLogger(__name__)
 
 
-class ETL(object):
+class IntakeToDissemination(object):
     def __init__(self, sac: SingleAuditChecklist) -> None:
         self.single_audit_checklist = sac
         self.report_id = sac.report_id
         audit_date = sac.general_information.get(
             "auditee_fiscal_period_start", datetime.now
         )
         self.audit_year = int(audit_date.split("-")[0])
+        self.loaded_objects = dict()
 
     def load_all(self):
-        load_methods = (
-            self.load_general,
-            self.load_secondary_auditor,
-            self.load_federal_award,
-            self.load_findings,
-            self.load_passthrough,
-            self.load_finding_texts,
-            self.load_captext,
-            # self.load_audit_info()  # TODO: Uncomment when SingleAuditChecklist adds audit_information
-        )
-        for load_method in load_methods:
+        load_methods = {
+            "Generals": self.load_general,
+            "SecondaryAuditors": self.load_secondary_auditor,
+            "FederalAwards": self.load_federal_award,
+            "Findings": self.load_findings,
+            "FindingTexts": self.load_finding_texts,
+            "Passthroughs": self.load_passthrough,
+            "CapTexts": self.load_captext,
+            "Notes": self.load_notes,
+            "Revisions": self.load_revision,
+        }
+        for _, load_method in load_methods.items():
             try:
+                # Each method writes results into self.loaded_objects
                 load_method()
             except KeyError as key_error:
                 logger.warning(
                     f"{type(key_error).__name__} in {load_method.__name__}: {key_error}"
                 )
+        return self.loaded_objects
+
+    def get_dissemination_objects(self):
+        return self.loaded_objects
+
+    def save_dissemination_objects(self):
+        for key in self.loaded_objects.keys():
+            for o in self.loaded_objects[key]:
+                o.save()
 
     def load_finding_texts(self):
         findings_text = self.single_audit_checklist.findings_text
         findings_text_entries = findings_text["FindingsText"]["findings_text_entries"]
```

Comment:
`findings_text["FindingsText"]["findings_text_entries"]` will raise a KeyError if `findings_text_entries` is missing, which could occur with empty submissions. Line 46 catches this error but using the get("findings_text_entries") method would be a more graceful way to handle the absence of this key and it would allow us to return an empty list [] instead of raising an exception